pg_cron--1.0--1.1.sql

```diff
@@ -0,0 +1,12 @@
+/* pg_cron--1.0--1.1.sql */
+
+DO $$ 
+BEGIN
+    BEGIN
+        ALTER TABLE cron.job ADD COLUMN active boolean not null default 'true';
+    EXCEPTION
+        WHEN duplicate_column THEN RAISE NOTICE 'column <active> already exists in <cron.job>.';
```

Comment:
I'm guessing you might be the only one who has this column already :). I'm slightly inclined to omit the exception handler because it seems like a situation that should not exist and permitting it might lead to other issues.

Be careful with using pg_cron after manually altering the `cron.job` table. As you can see from the source code it uses hard-coded indexes to look up columns.  If you drop a column in postgres, postgres will actually remember this column and its index forever. Therefore the index of the column necessarily its position in the table schema.